source/github-helpers/api.ts

```diff
@@ -54,7 +54,7 @@ export class RefinedGitHubAPIError extends Error {
 	}
 }
 
```

Comment:
What if you export a function instead?

```suggestion

export async function expectToken(): void {
	const {personalToken} = await settings;
	if (!personalToken) {
		throw new Error('Personal token required for this feature');
	}
}

```

And then call it in `api.v4` too

Reply:
👍

Reply:
I know why I didn't do that. 
Since I do not want to throw, I just want to log the error.
The reason it does not error out by all the other features, is because `logError` takes over.

https://github.com/sindresorhus/refined-github/blob/15f81938ad41c270cac360a99905cc265be3c6d9/source/features/index.tsx#L57

WDYT?

Reply:
You **do** want to throw, because that's how it's been setup. The errors are **caught** by `features.add` and displayed in the format that you copied from the console.

Reply:
It does not, it throws.